src/Converter.php

```diff
@@ -54,6 +57,31 @@ public function convert($text)
         $dom = new DOMDocument('1.0', 'UTF-8');
         $dom->loadHTML('<?xml encoding="UTF-8">' . $text);
 
-        return $this->factory->getConverterForTag($dom->documentElement)->getPdflibString();
+        return $this->
+            factory->
+            getConverterForTag($dom->documentElement)->
+            setLeftDelimiter($this->leftMacroDelimiter)->
+            setRightDelimiter($this->rightMacroDelimiter)->
```

Comment:
I'd rather set the delimiter on instantiating the converter than each time before I call `getPdfLibString`...